callisto/delivery/models.py

```diff
@@ -94,7 +94,7 @@ def _unpepper(peppered_report):
 class Report(models.Model):
     """The full text of a reported incident."""
     encrypted = models.BinaryField(blank=False)
-    owner = models.ForeignKey(settings.AUTH_USER_MODEL)
+    owner = models.ForeignKey(settings.AUTH_USER_MODEL, on_delete=models.CASCADE)
```

Comment:
we don't, `on_delete=models.CASCADE` is the default behavior, this line just makes it explicit